pytest/tests/loadtest/locust/common/base.py

```diff
@@ -322,6 +322,43 @@ def send_tx(self, tx: Transaction, locust_name: str) -> dict:
         self.request_event.fire(**meta)
         return meta
 
+    def send_tx_async(self, tx: Transaction, locust_name: str) -> dict:
```

Comment:
is this change part of this PR? I thought you had another PR for using async API

Reply:
No, it appeared here because of the base branch that I've used. Should be fixed now after the rebase.